Source/WebKit/WebProcess/WebPage/WKAccessibilityWebPageObjectIOS.mm

```diff
@@ -75,10 +75,16 @@ - (id)accessibilityHitTest:(NSPoint)point
     if (!m_page)
         return nil;
 
+    RefPtr focusedLocalFrame = [self focusedLocalFrame];
+
+#if ENABLE(UNIFIED_PDF)
```

Comment:
Better to go with `ENABLE(PDF_PLUGIN)`, since this is an iOS-only file anyway, and we plan to smoosh `UNIFIED_PDF` and `PDF_PLUGIN` together moving forward.